### PR TITLE
[GL-1604]Standardized formatting across existing images

### PR DESCRIPTION
**Pull request by wjdingman1**
This PR standardizes formatting and style across all the broad Dockerfiles, it also ensures that all images are labeled with their corresponding tool versions.

### CODE REVIEW

File: dockers/broad/zcall/Dockerfile [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
 WORKDIR /usr/gitc
 
 # Install dependencies
-RUN apk add --no-cache bash tini unzip curl && \
+RUN set eux; \
+        apk add --no-cache \
+        bash \
+        curl  \
+        tini \
+        unzip \
```
Comment (cheyenne-gold) [inline dockers/broad/zcall/Dockerfile:20]:
```suggestion
            bash \
            curl  \
            tini \
            unzip \
```

File: dockers/broad/verify_bam_id/Dockerfile [hunk 2]
```diff
@@ -1,38 +1,55 @@
...
-    rm -rf $GIT_HASH.zip VerifyBamID-$GIT_HASH && \
+    cmake; \
+    make; \
+    make test; \
     \
-    # Install tini
-    wget https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini -O /sbin/tini && \
-    chmod +x /sbin/tini && \
-    # Clean up cached files
+    cd ../../;\
```
Comment (cheyenne-gold) [inline dockers/broad/verify_bam_id/Dockerfile:44]:
```suggestion
    cd ../../; \
```

File: dockers/broad/arrays_picard_private/Dockerfile [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
+    curl -L ${ARTIFACTORY_URL}/${PICARD_PRIVATE_VERSION}/picard-private-${PICARD_PRIVATE_VERSION}.jar > picard-private.jar \
+    ; \
+# Download the gsutil install script
+    wget https://dl.google.com/dl/cloudsdk/channels/rapid/install_google_cloud_sdk.bash -O - | bash \
+    ; \ 
+# Set up Vault
+    curl -L https://releases.hashicorp.com/vault/1.0.2/vault_1.0.2_linux_amd64.zip > temp.zip; \
+    unzip temp.zip; \
+    rm temp.zip; \ 
     mv vault /usr/local/bin/
```
Comment (cheyenne-gold) [inline dockers/broad/arrays_picard_private/Dockerfile:37]: should there be a semicolon at the end of this line?

**Review by cheyenne-gold — APPROVED** [review]

File: dockers/broad/arrays_picard_private/Dockerfile (hunk 3 above)
Comment (wjdingman1) [reply dockers/broad/arrays_picard_private/Dockerfile:37]: dont need semicolon at the end of RUN command...it would the same thing as doing &&

**Review by khajoue2 — APPROVED** [review]
Looks good!

**Pull request merged**